app/fieldmapping.py

```diff
@@ -88,6 +88,20 @@ def map_employee_history_job_role_table_headers(
 
   return table_generation(mapping_entries)
 
+def map_full_address_details(employee_info):
+  complete_address = ''
```

Comment:
This line isn't needed at all. Don't initialize any variables that you're going to fully replace later.